functions/src/index.ts

```diff
@@ -1,27 +1,46 @@
-import {config, firestore as database, https} from 'firebase-functions';
+import {config, firestore as firestoreFromFunctions, https} from 'firebase-functions';
 import {Request, Response} from "express";
 import {createProbot, Options} from "probot";
 import {consoleStream, registerTasks, Tasks} from "./util";
-import {credential, firestore, initializeApp} from "firebase-admin";
+import {credential, firestore, initializeApp, app, database} from "firebase-admin";
 import {DocumentSnapshot} from "firebase-functions/lib/providers/firestore";
 import {EventContext} from "firebase-functions/lib/cloud-functions";
+import { HttpClient } from './http';
 
 let tasks: Tasks;
 let probotConfig: Options = config().probot;
+
+let sizeApp: app.App;
+
+const sizeConfig = require('../private/size-config.json');
+
 // Check if we are in Firebase or in development
 if(probotConfig) {
   // Init Firebase
   initializeApp();
 } else {
   // Use dev config
   probotConfig = require('../private/env.json');
+
   const serviceAccount = require("../private/firebase-key.json");
+  // default firebase account
   initializeApp({
     credential: credential.cert(serviceAccount)
   });
+  
+  const sizeServiceAccount = require("../private/firebase-key.json");
+  sizeApp = initializeApp({
+    credential: credential.cert(sizeServiceAccount),
+    databaseURL: sizeConfig.sizeDatabaseUrl,
+  }, 'sizeApp');
```

Comment:
this will work when you run firebase locally, but you also need to set the config for prod (in the "if" above)